### ISSUE TITLE
Modules do not support Amazon Linux

### ISSUE THREAD

**Issue by fields** [sensu/sensu-go-ansible#202]
When trying to install the sensu go agent on an Amazon Linux instance using the ansible modules, the following error occurs:

```
TASK [sensu.sensu_go.install : Include distro-specific vars (Amazon)] ****************************************************************************************************************************************************************
fatal: [xx.xx.xx.xx]: FAILED! => {"ansible_facts": {}, "ansible_included_var_files": [], "changed": false, "message": "Could not find or access 'Amazon.yml'
```

I was told that the modules only support the platforms listed here: https://docs.sensu.io/sensu-go/latest/platforms/

It would be nice if Amazon Linux was supported since it is the only available option for some AWS services, particularly ECS cluster hosts.

Thanks!

**Comment by tadeboro** [comment]
I was not involved in the initial decision making when the install role has been designed and thus I have no information as to *why* we selected RPMs/DEBs as an installation source. But looking back at that decision, it would probably be better to use binary-only distribution as a source of the installation.

Right now, we need to manually add each supported distribution to our installation role. If we were to use the bare binaries, our role would automatically work on all distributions that use one of the supported init systems (systemd, upstart, and good old init would cover a vast majority of the distributions). We would need to maintain our own copies of the service file definitions, but since those files do not change often, that should not be much of a problem.

The approach I described above would resolve this issue since binaries can run on Amazon Linux just fine and the init system used is systemd.

Maybe it would be worth adding this as another installation option? This way, Sensu Go users would be able to select the installation source: use RPMs/DEBs on distributions that have upstream packages or opt to use binary-only distribution on distros that are not officially supported but have a great chance of working.
